west.yml

```diff
@@ -21,6 +21,8 @@ manifest:
   remotes:
     - name: upstream
       url-base: https://github.com/zephyrproject-rtos
+    - name: rfc
+      url-base: https://github.com/laurenmurphyx64
```

Comment:
should this move to zephyr hosted repos?

Reply:
We will move it after this PR is reviewed and before it's merged :+1:

Reply:
@hakehuang @nashif Moved west.yml reference to the new [Zephyr-hosted TensorFlow repo](https://github.com/zephyrproject-rtos/tensorflow).